src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -304,6 +321,43 @@ func startNode(
 	return err
 }
 
+func checkBrokers(nodes []node) func() error {
+	return func() error {
+		addrs := make([]string, 0, len(nodes))
+		for _, n := range nodes {
+			addrs = append(addrs, n.addr)
+		}
+		client, err := kafka.InitClient(addrs...)
```

Comment:
hmm interesting. When I thought about this feature I thought we're going to use health check or something that's already baked in the docker image and then just read that the broker is healthy. That's not an option?

Reply:
Hmmm, I hadn't thought of that. Do you see any drawbacks to this approach though?

Reply:
no, I think this is fine. Potentially both can be in place. I was just wondering if you rejected that for some reason. Why I like the dockerfile approach is that with that solution, anyone using the dockerfile can benefit, not just rpk users

Reply:
But not a blocker for me, just wanted to bring it up :)